test/ae_database/encrypttable.ps1

```diff
@@ -0,0 +1,31 @@
+# Import the SqlServer module.
+Import-Module "SqlServer"
+
+# Connect to your database.
+$serverName = "yourServer"
+$databaseName = "AEDemo"
+$userName = "yourUsername";
+$password = "yourPassword";
+
+#For Windows Authentication
```

Comment:
Do all these files reference the same credentials? If so, consider creating a file only with credentials so that the other scripts can simply reference this file?